src/main/resources/templates/dev-hub/authentication.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
```

Comment:
There should be no blank line between the docctype and the html tag.

Reply:
Done